README.md

```diff
@@ -429,7 +470,7 @@ Returns **[ModelSchema](#modelschema)** model schema
 
 ## serializable
 
-[src/api/serializable.js:93-103](https://github.com/mobxjs/serializr/blob/b34674d96cb101c30739a497287d3c24ede31010/src/api/serializable.js#L93-L103 "Source code on GitHub")
+[src/api/serializable.js:93-103](https://github.com/evoye/serializr/blob/b7288a85d4a7e01d75cf9dfa9cc6ae677ebb3a67/src/api/serializable.js#L93-L103 "Source code on GitHub")
```

Comment:
Correct me if I'm wrong but I don't think this should change the address right?

Reply:
Hi Gordon, since this part of the file is autogenerated I do not have an influence here. The build generates the urls based on my forked repo. I assume based on the current build process you need to pull and then regenerate the docs and update version nr.

Reply:
This is resolved now. (I figured I can just solve this online in the commit review UI).